code/modules/shuttle/special.dm

```diff
@@ -262,40 +284,50 @@
 		payees[AM] += H.credits
 		counted_money += H
 
-	if(ishuman(AM))
-		var/mob/living/carbon/human/H = AM
-		if(H.get_bank_account())
-			var/datum/bank_account/account = H.get_bank_account()
+	if(payees[AM] < threshold)
+		var/armless
+		if(!ishuman(AM) && !istype(AM, /mob/living/simple_animal/slime))
```

Comment:
Is this intended to be not slime?

Reply:
Yeah. Slimes can't drag. Their drag proc is overridden. I wish there was an easier way to identify mobs that can't drag but aside from making a global list or changing how dragging is handled for all mobs I can't think of one.

Reply:
Got it.